test/docker.js

```diff
@@ -16,7 +17,9 @@ describe('containers', function () {
     async.waterfall([
       docker.createContainer.bind(docker, {}),
       function (container, cb) {
-        container.remove(cb);
+        var count = createCount(cb);
+        eventsStream.on('data', expectStatus('destroy', count.inc().next));
+        container.remove(count.inc().next);
```

Comment:
let's move this test down to the `events` `describe` block. this is testing "should create and delete a container", not that the event was fired (disconnect with the name and what's actually being tested)